system-monitor@paradoxxx.zero.gmail.com/schemas/org.gnome.shell.extensions.system-monitor.gschema.xml

```diff
@@ -458,6 +458,46 @@
 	  <default>false</default>
 	  <summary>Hide system battery icon</summary>
 	</key>
+  <key name="cpu-position" type="i">
```

Comment:
I reckon change the indentation of these new settings to match the others. Might as well fix up the indentation of the other elements too while we're at it.